src/__tests__/pages/api/auth/reset-password.test.ts

```diff
@@ -214,29 +191,14 @@ describe('/api/auth/reset-password: Failed', () => {
 
     await resetPasswordHandler(req, res);
 
-    expect(status.mock.calls[0]).toEqual([400]);
-    expect(json.mock.calls[0]).toEqual([
-      {
-        message: 'Email, password and new password are required'
-      }
-    ]);
-
-    req = {
-      body: {
-        email: email,
-        oldPassword: undefined,
-        newPassword: mockPassword()
-      } as resetPasswordProps
-    } as NextApiRequest;
+    req = { ...req, body: { ...req.body, oldPassword: undefined } as resetPasswordProps } as NextApiRequest;
 
     await resetPasswordHandler(req, res);
 
-    expect(status.mock.calls[0]).toEqual([400]);
-    expect(json.mock.calls[0]).toEqual([
-      {
-        message: 'Email, password and new password are required'
-      }
-    ]);
+    expect(status).toHaveBeenNthCalledWith(2, 400);
```

Comment:
What is the argument `2` supposed to be here

Reply:
I this test, I call the mock function 2 times, 1 with null para and 1 with undefined para. Therefore, we should expect this mock call function has been called 2 times and returns status 400 both times